src/components/Pagination/Pagination.stories.tsx

```diff
@@ -1,99 +1,30 @@
-import Icon from "../Icons/Icon";
-import { IconNames, IconRotationTypes } from "../Icons/IconTypes";
-import Label from "../Label/Label";
 import Pagination, { PaginationProps } from "./Pagination";
-import Select from "../Select/Select";
 
 export default {
     title: "Pagination",
     component: Pagination,
     decorators: [withDesign],
 };
 
-const selectOptions = (
-    <>
-        <Label htmlFor="paginationSelect" id={"paginationLabel"}>
-            {text("Select Label", "Page ")}
-        </Label>
-        <Select
-            name="Pagination Select"
-            id={"paginationSelect"}
-            isRequired={false}
-            ariaLabel="Pagination Label"
-            labelId={"paginationLabel"}
-            onBlur={action("blur")}
-            onChange={action("changed")}
-            selectedOption={"1 of 7"}
-        >
-            <option aria-selected={true}>
-                {text("Option 1 Text", "1 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 2 Text", "2 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 3 Text", "3 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 4 Text", "4 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 5 Text", "5 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 6 Text", "6 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 7 Text", "7 of 7")}
-            </option>
-        </Select>
-    </>
-);
-
-const PaginationTemplate = ({ children, ...args }) => (
-    <Pagination {...args}>{children}</Pagination>
+const PaginationTemplate: Story<PaginationProps> = (args) => (
+    <Pagination pageCount={args.pageCount} currentPage={args.currentPage} />
 );
 
 export const pagination = PaginationTemplate.bind({});
 
 pagination.args = {
-    previousPage: (
-        <Button
-            buttonType={ButtonTypes.Secondary}
-            onClick={action("Previous clicked")}
-        >
-            <Icon
-                name={IconNames.arrow}
-                decorative={true}
-                iconRotation={IconRotationTypes.rotate90}
-                modifiers={["small", "icon-left"]}
-            />
-            {text("Previous Button Label", "Previous")}
-        </Button>
-    ),
-    nextPage: (
-        <Button
-            buttonType={ButtonTypes.Secondary}
-            onClick={action("Next clicked")}
-        >
-            {text("Next Button Label", "Next")}
-            <Icon
-                name={IconNames.arrow}
-                decorative={true}
-                iconRotation={IconRotationTypes.rotate270}
-                modifiers={["small", "icon-right"]}
-            />
-        </Button>
-    ),
-    children: selectOptions,
+    pageCount: 100,
```

Comment:
For whatever reason I'm not able to click around to different pages on Storybook.

Reply:
I recently re-worked the component to remove having its own state, and instead added a new prop for the currentPage. There are some further details in the comment a little further above, but basically, after testing the Pagination component in the Location Finder, it seemed better to let the consuming application keep track.

Because the component no longer has its own state, you have to use the Storybook control for currentPage to simulate navigating through the Pagination.

I could look into adding a useState hook into the Storybook file so the example would be clickable. Should I give that a shot?

Reply:
> I recently re-worked the component to remove having its own state, and instead added a new prop for the currentPage. There are some further details in the comment a little further above, but basically, after testing the Pagination component in the Location Finder, it seemed better to let the consuming application keep track.
On the same page here—this is definitely the right direction for this component.

> Because the component no longer has its own state, you have to use the Storybook control for currentPage to simulate navigating through the Pagination.
> 
> I could look into adding a useState hook into the Storybook file so the example would be clickable. Should I give that a shot?
I would really appreciate you giving this a shot—I know that I wanted to understand how to click around in the component and see it in action. I suspect I won't be the only one confused.

Reply:
I was able to add the hooks to the Story, so that the Pagination component can be clicked in the example and it will update accordingly.